test-packages/static-app/ember-cli-build.js

```diff
@@ -21,6 +21,13 @@ module.exports = function (defaults) {
     staticAddonTrees: true,
     staticComponents: true,
     staticHelpers: true,
+    packagerOptions: {
+      // Setting devtool to source-map to ensure a failure if there's a regression on
+      // https://github.com/embroider-build/embroider/issues/725
+      webpackConfig: {
+        devtool: 'source-map',
+      },
+    },
```

Comment:
I wasn't sure of the best way to test this. It looks like there aren't unit tests for `html-placeholder.ts`, and I couldn't think of a better way other than "include the setting in one of the test apps, so the test suite will at least break if there's a regression". Let me know if there's a better way I could handle that.